javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+var $display = document.querySelector('.countdown-display');
+var count = 4;
+
+function countDown(event) {
+  count--;
+  if (count > 0) {
+    $display.textContent = count;
+  } else {
+    $display.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval();
```

Comment:
`clearInterval` doesn't clear anything unless you tell it which interval to clear: https://developer.mozilla.org/en-US/docs/Web/API/clearInterval